src/ZenithAngleCalc.jl

```diff
@@ -84,27 +81,28 @@ function distance_declination_hourangle(
     return d, δ, η_UTC
 end
 
+"""
+    instantaneous_zenith_angle(
+        (d, δ, η_UTC)::Tuple{FT, FT, FT},
+        longitude::FT,
+        latitude::FT,
+        param_set::IP.AIP,
+    ) where {FT}
+
+Returns the zenith angle, azimuthal angle and Earth-Sun distance
+at a particular longitude and latitude given Earth-Sun distance, declination angle, 
+and hour angle at longitude = 0.
```

Comment:
Good to clarify units here...
```suggestion
Returns the zenith angle, azimuthal angle (both in radians) and Earth-Sun distance
at a particular longitude and latitude (in degrees) given Earth-Sun distance, declination angle (in radians), 
and hour angle at longitude = 0.
```